c++/src/kj/compat/tls.c++

```diff
@@ -791,6 +791,13 @@ TlsContext::TlsContext(Options options) {
     throwOpensslError();
   }
 
+  // honor options.curveList
```

Comment:
[nit] Comment doesn't add any value.

Reply:
Keeping with the style around it.

Reply:
It's just a nit so it's not blocking but this is **very** widely considered to be bad style :)

Reply:
I agree that it's bad form, just would be a little weird in the context of the rest of the file.
https://github.com/capnproto/capnproto/pull/1811/files/85d4074ac5b5ec59527020b1c5b0bab1a6d7c6f0#diff-dac192e78e5c2b9147c63353b2f6f96e41d7101c9dbbe89698da0bf06fa49b47R789
https://github.com/capnproto/capnproto/pull/1811/files/85d4074ac5b5ec59527020b1c5b0bab1a6d7c6f0#diff-dac192e78e5c2b9147c63353b2f6f96e41d7101c9dbbe89698da0bf06fa49b47L794
https://github.com/capnproto/capnproto/pull/1811/files/85d4074ac5b5ec59527020b1c5b0bab1a6d7c6f0#diff-dac192e78e5c2b9147c63353b2f6f96e41d7101c9dbbe89698da0bf06fa49b47L766